python/tests/kat/t_tracing.py

```diff
@@ -144,8 +144,8 @@ def check(self):
         assert self.results[101].backend
         assert self.results[101].backend.name == "raw"
 
-        tracelist = {x: True for x in self.results[101].backend.response}
-
+        tracelist = set(x for x in self.results[101].backend.response)
+        print(f"tracelist = {tracelist}")
```

Comment:
Is this going to be too noisy? how big does the set end up being?

Reply:
The size depends on the test; I think the test with the largest list has 3 items in it.